data/kahet/aberrant outfits.txt

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Weapons:
+
+
+outfit "Aberrant Spiker"
```

Comment:
400 speed, so fast ships can probably outrun them as long as they're not trying to do anything else.
12000 effective range is probably excessively long.
Ammo presumably lasts for 60 seconds (60 ammo capacity, 1 sec reload) - towards the upper end for homing weapons. Although as far as I can see you haven't made this actually use anything as ammunition yet.
During that time, this 15 ton weapon is putting out 1000 DPS, giving it almost double the Finisher's DPS per ton.

If this is supposed to be ammo limited, you need to add the ammo requirement and the damage needs to go down. If it's not, the damage needs to go down a lot.